README.md

```diff
@@ -1,2 +1,153 @@
-# TDD
-In this project you will implement a class with some methods, but you will do it by doing test-driven development (TDD).
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>  
+  <h2><b>Test-driven development (TDD)</b></h2>
+<!--  -->
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<br>
+
+# 📖 TDD <a name="Test-driven development (TDD)"></a>
+
+> Test-driven development (TDD) In this project there is implemented a class with some methods, write tests first and then the code
+>
+> - Add new test
+> - Failing test
+> - Add code
+> - Passing test
+>   <br>
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+- TDD
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+<br>
+
+### Key Features <a name="key-features"></a>
+
+- TDD
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a> -->
+<!-- >[Live Demo](https://airmetrics.onrender.com) -->
+
+<!-- > [Video Presentation](https://www.loom.com/share/2ad53b4776e94f43bb9c464cf7a63c5e?sid=014845e4-9b2d-4449-9e89-59e30a0f64d4) -->
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. Interactive Ruby shell (IRB).
+2. Code Editor.
+3. Tests
+
+### Setup
+
+> To clone this repository to your desired folder, run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/TamaraDimov/TDD.git
+```
+
+### Install
+
+> Run this command:
+
+```sh
+  cd TDD
+  bundle install
+  gem install rspec
+```
+
+### Usage
+
+> Run the code
+
+### Run tests
+
+> comming soon
+
```

Comment:
- [ ] To make your documentation looks more professional and descriptive, I think it would be better if you update the content of the current section with the appropriate content information i.e. the command of how to run your tests(`rspec spec/ `)